src/pages/faq.tsx

```diff
@@ -1,11 +1,195 @@
+    {stateAction[3][0] ? (
+    <div className="pl-[20px] h-[110px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
+    <p className="text-white">{t.faq.table3text4} <a className="text-cerise" href="mailto:sales@ddagen.se">sales@ddagen.se</a></p>
+    </div>) : null}
+</div>)
+}
+
+function Table4(stateAction: Array<[boolean, React.Dispatch<React.SetStateAction<boolean>>]>){
+  const t = useLocale()
+  return(
+  <div className="mt-[50px]">
+
+    <button onClick={() => stateAction[0][1](!stateAction[0][0])} id="accordion" className="pl-[20px] text-left text-white h-[130px] sm:h-[55px] w-full bg-slate-50 bg-opacity-20 border-[3px] border-cerise"> {t.faq.table4row1}</button>
+    {stateAction[0][0] ? (
+    <div className="pl-[20px] h-[270px] sm:h-[100px] bg-gray bg-opacity-50 border-[1px] border-cerise flex items-center ">
+    <p className="text-white">{t.faq.table4text1} <a className="text-cerise" href="mailto:alexandre.moch@ddagen.se">alexandre.moch@ddagen.se</a></p>
+    </div>) : null}
+    
+</div>)
+}
 
 export default function Faq() {
-    const t = useLocale();
+  const t = useLocale();
+  const slides = [
+    {
+      url: "/img/slide1.png",
+    },
+    {
+      url: "/img/slide2.png",
+    },
+    {
+      url: "/img/slide3.png",
+    },
+  ];
 
-    return (
-        <div>
-            <p>FAQ</p>
+  const tables = [Table1([useState(false), useState(false), useState(false)]), Table2([useState(false), useState(false)]), Table3([useState(false), useState(false), 
+  useState(false), useState(false)]), Table4([useState(false)])]
+  
+  const [currentTable, setCurrentTable] = useState(0);
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  const Tst = tables[0]
+
+  return (
+    <div className="py-[200px] font-['NeueHaasDisplayRoman'] flex flex-col items-center">
+
+      {/*Header*/}
+      <div className="w-full ">
+        <p className=" text-center text-cerise text-[50px] sm:text-[100px] font-['NeueHaasDisplayRoman'] w-full">
+          {" "}
+          EXHIBITOR
+        </p>
+        <p className=" text-center text-cerise text-[50px] sm:text-[100px] font-['NeueHaasDisplayRoman'] w-full">
+          {" "}
+          FAQ
+        </p>
+      </div>
+      {/*Header*/}
+      
+      {/*Carousel*/}
+      <div className="mt-[150px]">
+        <div id="carousel"
+            className="px-[0px] sm:px-[20px] "
+            > <img src={slides[currentSlide].url} className="min-w-[200px]"></img>
+        </div>
+        <div className="flex justify-center mt-[50px]">
+          {slides.map((slide, index) => (
+            <button
+              key={index}
+              className={`mx-2 w-5 h-5 bg-gray rounded-full cursor-pointer ${
+                currentSlide === index ? "bg-slate-300" : ""
+              }`}
+              onClick={() => setCurrentSlide(index)}
+            ></button>
+          ))}
         </div>
-    )
```

Comment:
Looks great overall. Just a bitt too big on my screen. 
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/66337956/229503242-ffc81a28-78f6-4847-b226-0d82426dadc0.png">

Reply:
It is now smaller.